database/config.py

```diff
@@ -1,9 +1,15 @@
+import os
+import urlparse
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from sqlalchemy.ext.declarative import declarative_base
 
 
-engine = create_engine('mysql+mysqldb://root:root@localhost/ouija2', echo=True)
+DBURL = urlparse.urlparse(os.environ["DATABASE_URL"])
```

Comment:
please add a comment in the source code and readme about this environment variable and possible values.  We can make it smarter in the future.